assets/src/components/button/style.scss

```diff
@@ -0,0 +1,83 @@
+/**
+ * Button styles,
+ */
+
+
+.newspack-button {
+	.components-button {
+	    padding: 0 20px 1px;
+		text-align: center;
+
+		font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen-Sans", "Ubuntu", "Cantarell", "Helvetica Neue", sans-serif;
+		font-size: 14px;
+		line-height: 18px;
+		font-weight: 500;
+		color: $muriel-gray-700;
+		text-shadow: none;
+
+		&.is-button {
+			background-color: $muriel-white;
+			border: none;
+			border-radius: 3px;
+			cursor: pointer;
+			height: 40px;
+			max-width: 360px;
+			max-height: 40px;
+			margin-top: 13px;
```

Comment:
I think we may have margin and padding mixed up here. Let's do the following (all within `&.is-button`);

```
padding: 13px 0 9px 0;
margin: 14px auto;
```